src/hotspot/share/runtime/thread.hpp

```diff
@@ -115,16 +115,16 @@ class Thread: public ThreadShadow {
 
   // On AArch64, the high order 32 bits are used by a "patching epoch" number
   // which reflects if this thread has executed the required fences, after
-  // an nmethod gets disarmed. The low order 32 bit denote the disarm value.
-  uint64_t _nmethod_disarm_value;
+  // an nmethod gets disarmed. The low order 32 bit denote the disarmed value.
```

Comment:
Nit:
I think this should be:
"The low order 32 bits denote the disarmed value."
instead of:
"The low order 32 bit denote the disarmed value."

Reply:
Yes, you are right, thanks.